x-pack/plugins/cases/public/components/all_cases/use_bulk_actions.test.tsx

```diff
@@ -348,6 +361,110 @@ describe('useBulkActions', () => {
     });
   });
 
+  describe('Flyouts', () => {
+    it('change the tags of the case', async () => {
+      const updateCasesSpy = jest.spyOn(api, 'updateCases');
+
+      const { result, waitFor: waitForHook } = renderHook(
+        () => useBulkActions({ onAction, onActionSuccess, selectedCases: [basicCase] }),
+        {
+          wrapper: appMockRender.AppWrapper,
+        }
+      );
+
+      let flyouts = result.current.flyouts;
+      const panels = result.current.panels;
+
+      const res = appMockRender.render(
+        <>
+          <EuiContextMenu initialPanelId={0} panels={panels} />
+          {flyouts}
+        </>
+      );
+
+      act(() => {
```

Comment:
I think [the following](https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#wrapping-things-in-act-unnecessarily) applies here. `user-event` is built on top of `fireEvent` and
 
> fireEvent are already wrapped in act!

Now, I tried removing all instances in this file of `userEvent.click` surrounded by `act(() => {` and ran the tests to see if we got any warnings and we do, a bunch of them.

> Warning: It looks like you're using the wrong act() around your test interactions.
    Be sure to use the matching version of act() corresponding to your renderer:

But I had seen this before and it is an easy fix.

You can just change line 12 to be `import { act, renderHook } from '@testing-library/react-hooks/dom';` and they all go away. 

[See also.](https://react-hooks-testing-library.com/installation#being-specific) I think the act used in `@testing-library/user-event` is different than the one the `react-hooks` lib auto-detects so we need to force it to be the right one.

The common above also applies to `x-pack/plugins/cases/public/components/all_cases/use_actions.test.tsx`.

Reply:
Awesome! Thanks, @adcoelho.

Reply:
Nice, didn't know we need to verify which act is used 😄